src/core/server/index.ts

```diff
@@ -396,14 +402,7 @@ export type {
   SavedObjectsImporter,
   SavedObjectsImportError,
 } from './saved_objects';
-export {
-  SavedObjectsClient,
-  SavedObjectsErrorHelpers,
-  SavedObjectsSerializer,
-  SavedObjectTypeRegistry,
-  SavedObjectsUtils,
-  mergeSavedObjectMigrationMaps,
-} from './saved_objects';
+export { SavedObjectsClient } from './saved_objects';
```

Comment:
I need to check if we still need to export `SavedObjectsClient` and `SavedObjectTypeRegistry` from the entrypoint, but this will be done in the next PR or in the follow-up cleanup.